### ISSUE TITLE
Examples build failing.

### ISSUE THREAD

**Issue by alejandronanez** [shoutem/ui#172]
Hey there, I'm following the steps under: http://shoutem.github.io/docs/ui-toolkit/introduction

```shell
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:40:28: error: redefinition of 'RCTLogLevel'
typedef NS_ENUM(NSInteger, RCTLogLevel) {
                           ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:40:28: note: previous definition is here
typedef NS_ENUM(NSInteger, RCTLogLevel) {
                           ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:41:3: error: redefinition of enumerator 'RCTLogLevelTrace'
  RCTLogLevelTrace = 0,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:41:3: note: previous definition is here
  RCTLogLevelTrace = 0,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:42:3: error: redefinition of enumerator 'RCTLogLevelInfo'
  RCTLogLevelInfo = 1,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:42:3: note: previous definition is here
  RCTLogLevelInfo = 1,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:43:3: error: redefinition of enumerator 'RCTLogLevelWarning'
  RCTLogLevelWarning = 2,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:43:3: note: previous definition is here
  RCTLogLevelWarning = 2,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:44:3: error: redefinition of enumerator 'RCTLogLevelError'
  RCTLogLevelError = 3,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:44:3: note: previous definition is here
  RCTLogLevelError = 3,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:45:3: error: redefinition of enumerator 'RCTLogLevelFatal'
  RCTLogLevelFatal = 4
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:45:3: note: previous definition is here
  RCTLogLevelFatal = 4
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:51:28: error: redefinition of 'RCTLogSource'
typedef NS_ENUM(NSInteger, RCTLogSource) {
                           ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:51:28: note: previous definition is here
typedef NS_ENUM(NSInteger, RCTLogSource) {
                           ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:52:3: error: redefinition of enumerator 'RCTLogSourceNative'
  RCTLogSourceNative = 1,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:52:3: note: previous definition is here
  RCTLogSourceNative = 1,
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:10:
In file included from ../react-native/React/Base/RCTConvert.h:17:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTLog.h:53:3: error: redefinition of enumerator 'RCTLogSourceJavaScript'
  RCTLogSourceJavaScript = 2
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:10:
../react-native/React/Base/RCTLog.h:53:3: note: previous definition is here
  RCTLogSourceJavaScript = 2
  ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:11:
I
n file included from ../react-native/React/Base/RCTBridge.h:13:
/Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTBridgeModule.h:55:11: warning: duplicate protocol definition of 'RCTBridgeModule' is ignored
@protocol RCTBridgeModule <NSObject>
          ^
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.m:9:
In file included from /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/node_modules/react-native-vector-icons/RNVectorIconsManager/RNVectorIconsManager.h:9:
../react-native/React/Base/RCTBridgeModule.h:55:11: note: previous definition is here
@protocol RCTBridgeModule <NSObject>
          ^
1 warning and 9 errors generated.



** BUILD FAILED **


The following build commands failed:

	CompileC /Users/alejandro/Code/projects/goskate/ui-kits/shoutem/ios/build/Build/Intermediates/RNVectorIcons.build/Debug-iphonesimulator/RNVectorIcons.build/Objects-normal/x86_64/RNVectorIconsManager.o RNVectorIconsManager/RNVectorIconsManager.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)

Installing build/Build/Products/Debug-iphonesimulator/shoutem.app
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=2):
Failed to install the requested application
An application bundle was not found at the provided path.
Provide a valid path to the desired application bundle.
Print: Entry, ":CFBundleIdentifier", Does Not Exist

Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/shoutem.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist
```

Also my `index.ios.js` look like:
```js
/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 * @flow
 */
 import React, { Component } from 'react';
 import { AppRegistry } from 'react-native';
 import { Examples } from '@shoutem/ui';

export default class shoutem extends Component {
  render() {
    return (
      <Examples />
    );
  }
}

AppRegistry.registerComponent('shoutem', () => shoutem);
```

And got this bunch of errors. Any suggestion is appreciated.

**Comment by TimBroddin** [comment]
I've got the same problem

**Comment by jolic** [comment]
You need to update react-native-linear-gradient:
```js
npm install react-native-linear-gradient@2.0.0
```

Remove ```BVLinearGradient.xcodeproj``` from the Libraries of the project and add ```./node_modules/react-native-linear-gradient/BVLinearGradient.xcodeproj``` again to the Libraries in your xcodeproj.

I've seen a couple of problems to get the example to work and there's no response from the developers.
I assume they don't have an interest to support the community as their interest is to get customers to use their Webtool to earn money.

All the example are really nice but are only usable through their Webtool:
https://github.com/shoutem/extensions/blob/master/LICENSE

So I've decided for me to use http://nativebase.io/ but there are also like http://www.xinthink.com/react-native-material-kit/

**Comment by alejandronanez** [comment]
What a bummer :(

To complement your response @jolic I'd add https://avocode.com/nachos-ui/ to that list too.

Thanks for your help!

**Comment by SoHotSoup** [comment]
Hey guys, I will try to address this in next few days. I believe it's connected to our issues with RN0.40.0. I wouldn't say that we don't have a interest in supporting the community, and I can promise you that new release would bring some cool features to the community.

**Comment by alejandronanez** [comment]
Hey @SoHotSoup, it's good to hear something from a member of the project!
So any date for the new release?

**Comment by SoHotSoup** [comment]
I hope by the end of the week

**Comment by andrelevi** [comment]
Looking forward to the new release!

**Comment by hanpanpan200** [comment]
Hi @alejandronanez 

From your error message, we can know that [react-native-vector-icons](https://github.com/oblador/react-native-vector-icons) caused that error.

Then there is a [similar react-native-vector-icons issue](https://github.com/oblador/react-native-vector-icons/issues/375) 

So from the issue I think that maybe:

The `react-native` version you're using in the project is `>0.40`, but the version of `react-native-vector-icons` that you installed is '<0.4.0'.

I resolved the problem by (I'm using [yarn](https://yarnpkg.com/en/)):

```
yarn upgrade react-native-vector-icons@0.4
react-native link react-native-vector-icons 
```

If you're using npm, I think you can try:

```
npm update react-native-vector-icons@0.4
react-native link react-native-vector-icons 
```
Hope that can be helpful.

I hope the author of this library can add the message above to `Trouble Shot` section, I think many people are facing same problems since react-native have released `0.40.0` .

**Comment by SoHotSoup** [comment]
Fixed with latest release.
